services/cd-service/pkg/repository/repository.go

```diff
@@ -894,6 +899,25 @@ func (s *State) GetEnvironmentApplicationLocks(environment, application string)
 	}
 }
 
+func (s *State) GetDeploymentMetaData(environment, application string) (deployAuthor, deployTime string, err error) {
+	base := s.Filesystem.Join("environments", environment, "applications", application, "version", "environments", environment)
+	logs, err := readFile(s.Filesystem, s.Filesystem.Join(base, "deployment_metadata.yaml"))
```

Comment:
it's not yaml though, is it?
please remove the yaml suffix. Maybe replace with ".csv"